defaults/main.yml

```diff
@@ -267,12 +268,12 @@ vault_azurekeyvault_backend: vault_seal_azurekeyvault.j2
 # gcpkms seal
 vault_gkms: false
 vault_backend_gkms: vault_seal_gcpkms.j2
-vault_gkms_project: ''
-vault_gkms_credentials_src_file: ''
-vault_gkms_credentials: '/home/vault/vault-kms.json'
-vault_gkms_region: 'global'
-vault_gkms_key_ring: 'vault'
-vault_gkms_crypto_key: 'vault_key'
+vault_gkms_project: ""
```

Comment:
use single quotes when strings are 'literal values' or values with potential [gotchas](https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#gotchas)!

Empty strings should be single-quoted IMHO.